code/__DEFINES/preferences.dm

```diff
@@ -1,7 +1,7 @@
 
 //Preference toggles
 #define SOUND_ADMINHELP	1
-#define SOUND_MIDI		2
+#define UNUSED		2

```

Comment:
you'd have to make an update proc to discard the value from savefiles, and then up the savefile version.